operator/pkg/helmreconciler/prune.go

```diff
@@ -174,22 +171,20 @@ func (h *HelmReconciler) GetPrunedResources(revision string, purge bool) ([]*uns
 		if err != nil {
 			return usList, resources, err
 		}
-		if !purge {
+		if !includeClusterResources {
```

Comment:
nit: negation is unnecessary here i.e. if includeClusterResources {} - reverse the condition blocks below.